cmd/terraform_commands.go

```diff
@@ -18,6 +21,12 @@ func getTerraformCommands() []*cobra.Command {
 			Annotations: map[string]string{
 				"nativeCommand": "true",
 			},
+			PostRunE: func(cmd *cobra.Command, args []string) error {
+				fmt.Println("planPostRun")
+
+				hooks := cmd.Context().Value("hooks").(*h.Hooks)
+				return hooks.RunAll(h.AfterTerraformPlan, cmd, args)
```

Comment:
maybe we can use some parsing so that we can add hooks for all terraform commands and not just plan and apply?

Reply:
Eventually, I agree. But following the `YAGNI` principle, we only need certain hooks for the `store` functionality.